.gitignore

```diff
@@ -3,4 +3,5 @@ target
 /src/version.rs
 tests/gir-files
 tests/sys/*-sys/**
+tests/sys/sys_build/**
 !tests/sys/*-sys/Cargo.toml
```

Comment:
No, why even ignore the whole crate if you only have to ignore its `Cargo.lock` and `target`.